<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridSettingsForm</class>
 <widget class="QWidget" name="GridSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>187</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Видимость блоков</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="XGroupBox">
     <property name="title">
      <string>X-слой</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="XListView"/>
      </item>
      <item>
       <widget class="QPushButton" name="XClearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="XCheckAllButton">
        <property name="text">
         <string>Отметить все</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="YGroupBox">
     <property name="title">
      <string>Y-слой</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="YListView"/>
      </item>
      <item>
       <widget class="QPushButton" name="YClearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="YCheckAllButton">
        <property name="text">
         <string>Отметить все</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ZGroupBox">
     <property name="title">
      <string>Z-слой</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListView" name="ZListView"/>
      </item>
      <item>
       <widget class="QPushButton" name="ZClearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ZCheckAllButton">
        <property name="text">
         <string>Отметить все</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
